docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md

```diff
@@ -55,49 +57,12 @@ Get transfer/s Participant/s details.
 
 #### Description
 
-Get transfer/s Party/s details.
+Where a participant DFSP queries another participant DFSP for the details of a Party which the second DFSP holds, this UC is used to validate the request and provide the requested Party data to the requesting DFSP.
```

Comment:
If we opt for `Use Case` vs `UC`, we also need to change it here to be consistent throughout.

Reply:
It is defined as a Common Term --> https://ref-arch-docs.moja-lab.live/refarch/boundedContexts/intro.html

So I am ok to keep it as long as we are consistently using it.